Gemfile

```diff
@@ -1,18 +1,10 @@
-source "http://rubygems.org"
+# A sample Gemfile
+source "https://rubygems.org"
 
-if ENV.key?('PUPPET_VERSION')
-  puppetversion = "= #{ENV['PUPPET_VERSION']}"
-else
-  puppetversion = ['~> 2.7']
-end
-
-gem "rake"
-gem "puppet", puppetversion
-gem "puppet-lint"
-gem "rspec-puppet"
-gem "rspec-system-puppet"
-gem "puppetlabs_spec_helper"
-gem "puppet-syntax"
```

Comment:
Why remove the syntax checker? I think you have reimplemented something similar in the Rakefile?